src/components/Picklist/styled/input.js

```diff
@@ -43,6 +43,7 @@ const PickerInput = attachThemeAttrs(styled(StyledInput))`
         box-shadow: none;
         background-color: transparent;
         border: 2px solid transparent;
+        pointer-events: none;
```

Comment:
The problem with this solution is that it changes the course to the hand that indicates some action and the course should be the text